### PR TITLE
Add initial notebook.

### PR DESCRIPTION
**Pull request by scottdangelo**

### CODE REVIEW

File: notebooks/Telco-customer-churn-ICP4D.ipynb [hunk 1]
```diff
@@ -0,0 +1,608 @@
...
+   "source": [
+    "spark_df.printSchema()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 3.3 Use StringIndexer to encodes a string column of labels to a column of label indices"
+   ]
```
Comment (rhagarty) [inline notebooks/Telco-customer-churn-ICP4D.ipynb:407]: encodes/encode

File: notebooks/Telco-customer-churn-ICP4D.ipynb [hunk 2]
```diff
@@ -0,0 +1,608 @@
...
+   "outputs": [],
+   "source": [
+    "df = df1"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 2.1 Drop CustomerID feature (column)"
```
Comment (rhagarty) [inline notebooks/Telco-customer-churn-ICP4D.ipynb:77]: before deleting column, maybe first print the table to show columns

**Review by rhagarty — CHANGES_REQUESTED** [review]

File: notebooks/Telco-customer-churn-ICP4D.ipynb (hunk 1 above)
Comment (scottdangelo) [reply notebooks/Telco-customer-churn-ICP4D.ipynb:407]: fixed

File: notebooks/Telco-customer-churn-ICP4D.ipynb (hunk 2 above)
Comment (scottdangelo) [reply notebooks/Telco-customer-churn-ICP4D.ipynb:77]: When you insert the data, it adds:
`df2.head()`
So the table structure with columns is shown there.

**Review by rhagarty — APPROVED** [review]

**Pull request merged**